docs/examples/changing-the-src-dir.md

```diff
@@ -0,0 +1,79 @@
+---
+description: |
+  Change the source directory to effectively manage your project.
+---
+
+When you initialize a project with `deno run -A -r https://fresh.deno.dev`,
+you'll end up with a project like the following:
+
+```
+.
+├── README.md
+├── components
+│   └── Button.tsx
+├── deno.json
+├── dev.ts
+├── fresh.gen.ts
+├── islands
+│   └── Counter.tsx
+├── main.ts
+├── routes
+│   ├── [name].tsx
+│   ├── api
+│   │   └── joke.ts
+│   └── index.tsx
+└── static
+    ├── favicon.ico
+    └── logo.svg
+```
+
+## Using a `src` directory
+
+If you'd like your code to live in an `src` directory (or any other directory of
+your choosing), then you'll need to do the following things:
+
+1. Move all your files, except `deno.json` and `README.md`, to the `src`
+   directory.
+2. Modify the `start` task in `deno.json` to point to the new directory.
+
+Here's what the diff of `deno.json` looks like:
+
+```diff
+ {
+   "lock": false,
+   "tasks": {
+-    "start": "deno run -A --watch=static/,routes/ dev.ts"
++    "start": "deno run -A --watch=src/static/,src/routes/ src/dev.ts"
+   },
+   "imports": {
+     "$fresh/": "file:///Users/reed/code/fresh/",
+```
+
+(An astute observer would note that I actually generated this test project from
+my local build of Fresh, via `deno run -A -r ~/code/fresh/init.ts`).
+
+The final result of running `tree` looks like this:
```

Comment:
```suggestion
The resulting file structure looks like this:
```
Ah, my bad missed this previously.

Reply:
Hmmm, ok, I guess, but where do these little snippets go? Once upon a time I didn't know how people were producing these directory structures, so I think it's nice to include a reference to `tree`. Additionally, as I was doing this, it wasn't immediately obvious how to generate a project with my local build. I thought it would be good to include that too.